app/views/collections/show.html.slim

```diff
@@ -89,17 +89,25 @@
               .input-group
                 = text_field_tag 'user', nil, form: 'share', class: 'form-control'
                 = submit_tag t('.button.share'), form: 'share', class: 'btn btn-light'
-          .dropup.btn-group
+          .dropdown.btn-group
             = button_tag class: 'btn btn-important dropdown-toggle', 'data-bs-toggle': 'dropdown' do
-              => t('common.button.export')
+              => t('common.button.download_zip')
               span.caret
             .scrollable.dropdown-menu role='menu'
+              .dropdown-header = "#{t('common.button.available_versions')}: "
               - if @collection.tasks.empty?
                 a.dropdown-item.disabled-cursor title=t('.download_no_exercises')
                   = t('common.button.download_zip')
```

Comment:
This is not yet working:

- it looks off
- the dropdown entry is clickable
- the tooltip is not a correct Bootstrap tooltip

<img width="427" alt="Bildschirmfoto 2024-08-09 um 17 13 30" src="https://github.com/user-attachments/assets/69ddccfb-4db5-45a8-b2c7-4062c000cc4b">